packages/flutter/lib/src/material/data_table.dart

```diff
@@ -357,6 +357,11 @@ class DataCell {
 /// [PaginatedDataTable] which automatically splits the data into
 /// multiple pages.
 ///
+/// ## Performance considerations when wrapping [DataTable] with [SingleChildScrollView]
```

Comment:
@Piinks Do you have any suggestions for this section?

Reply:
Just the comment below, we should provide an alternative recommendation. :)

Reply:
So, suggest using `PaginatedDataTable` wherever possible?
Or is there a better alternative

Reply:
> we should recommend PaginatedDataTable to split it up, or using a CustomScrollView.

I think those two would be the best recommendation for now. :)

Reply:
Now that TableRow is merged to master, should that be added as a recommendation?

Reply:
Let's just add the previously mentioned recommendations for now so we can land this PR. We'll likely do a sweep overall to see where we should be mentioning the TableView.